### DISCUSSION TITLE
WingetUI 3.0.0 Alpha 2

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#1818] [Show and tell]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.0.0-alpha2/total?style=for-the-badge&color=red)

# Main changes since Alpha 1
 - Upgraded .NET 6.0 to .NET 8.0. The binary size has been decreased in 60mb
 - Added support for the background api, featuring
   - Compatibility with receiving shared packages
   - Compatibility with WingetUI Widgets
 - WingetUI won't crash with file pickers anymore when ran as administrator
 - Added the built-in autoupdater
 - Minor scoop, chocolatey and winget improvements
 - Minor interface improvements
 - Other bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/WingetUI/releases/tag/3.0.0-alpha2'>WingetUI 3.0.0 Alpha 2</a>.</em>

**Comment by dcog989** [comment]
`Settings... General Preferences... Update WingetUI automatically`

Does this mean update on manual start? Or is it installing a service? Or adding to Windows' start-up apps? Same question for 'Check for package updates', 'Update packages automatically'.

I think this needs clarifying in the UI but not sure what to suggest.

**Reply by marticliment** [reply]
WingetUI runs automatically on system start, and stays on the tray. When WingetUI is on the tray, it will autoupdate apps and update itself if applicable

**Reply by dcog989** [reply]
But I don't have 'Autostart WingetUI...' checked and it's unclear (to me at least!) whether the update settings in General Preferences function *separately* from the Autostart option. Or, do I need the update settings checked in order for them to happen when I open the app manually?

**Reply by marticliment** [reply]
I will clarify this
